deepmd/pt/utils/stat.py

```diff
@@ -476,19 +480,23 @@ def compute_output_stats_global(
     std_atom_e = {}
     for kk in keys:
         if kk in stats_input:
-            if atomic_output is not None and atomic_output.get_data()[kk].intensive:
-                task_dim = stats_input[kk].shape[1]
-                assert merged_natoms[kk].shape == (nf[kk], ntypes)
-                stats_input[kk] = (
-                    merged_natoms[kk].sum(axis=1).reshape(-1, 1) * stats_input[kk]
+            if property_name is not None:
+                assert len(keys) == 1
+                bias_atom_e["property"], std_atom_e["property"] = (
+                    compute_stats_property(
```

Comment:
1. bad name.  may be something like compute_stats_do_not_distinguish_types. 
2. you may let the fitting net to tell what type of statistics does it want. 
3. how do you distinguish the stats of intensive and extensive properties?